product_supplierinfo_discount/README.rst

```diff
@@ -47,6 +47,7 @@ Contributors
 ------------
 
 * Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+* Jonathan Nemry <jonathan.nemry@acsone.eu>
```

Comment:
up:
"This module allows to input a discount"
and
"The discount appears explicitly in purchase orders instead of being directly discounted in price."